Reference/Configuration/RichTextEditorSettings/index.md

```diff
@@ -87,3 +87,21 @@ Remove default ```<p>``` tag
     }
   }
 ```
+
+Add custom styles to the formats dropdown list.
+:::tip
+Use a text editor to find and replace `\"` with `"`. This will allow you to edit the json file in an easier to read format. Don't forget to add the `\` back in when you are ready to paste the code back into your app settings.
```

Comment:
```suggestion
Use a text editor to find and replace `\"` with `"`. This will allow you to edit the JSON file in an easier-to-read format. Don't forget to add the `\` back in when you are ready to paste the code back into your `appsettings.json` file.
```